src/audio-worklet/free-queue/examples/hello-webassembly/sink-processor.js

```diff
@@ -0,0 +1,42 @@
+import FreeQueue from '../../src/free-queue.js';
+
+const RENDER_QUANTUM = 128
+
+/**
+ * A simple AudioWorkletProcessor node that will act as sink.
+ * It will pull out audio from queue and render it.
+ * 
+ * @class BasicProcessor
+ * @extends AudioWorkletProcessor
+ */
+class SinkProcessor extends AudioWorkletProcessor {
+  constructor(options) {
+    super();
+    /**
+     * Initialize the Queue.
```

Comment:
The block comments are commonly used for JSDoc generator. Please use multiple `//` to make multi-line comments.